starfish/image/_single.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import collections
+from typing import Tuple
+
+from slicedimage import ImageFormat
+from slicedimage.io import resolve_url
+
+from ._base import ImageBase
+
+
+class SingleImage(ImageBase):
+    def __init__(self, data):
+        self._data = data
+
+    @classmethod
+    def from_url(cls, relativeurl: str, baseurl: str):
```

Comment:
This seems like it could be an API entrypoint. Can we get a docstring for this with an example?

Reply:
#197